toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -359,10 +359,15 @@ void PreferencesPopup::onDropdownShortcutsCycleOptionsChanged(int index) {
 void PreferencesPopup::rebuilldFontStyleList() {
   TFontManager *instance = TFontManager::instance();
   std::vector<std::wstring> typefaces;
-  QString font = m_interfaceFont->currentText();
-  instance->loadFontNames();
-  instance->setFamily(font.toStdWString());
-  instance->getAllTypefaces(typefaces);
+  QString font  = m_interfaceFont->currentText();
+  QString style = m_interfaceFontStyle->currentText();
```

Comment:
Is this variable `style` used in somewhere?

Reply:
Originally I was going to populate `typefaces` with this value if  a TFontCreationError was caught, but decided to just leave it blank and forgot to remove the variable. It can be removed.

Reply:
On second thought, I will populate it with the prefered font style even if it is invalid.